Dockerfile

```diff
@@ -80,6 +80,7 @@ COPY --from=builder /app/app/static app/static
 COPY --from=builder /app/app/pkg app/pkg
 COPY --from=builder /app/target/release/lldap /app/target/release/lldap_migration_tool /app/target/release/lldap_set_password ./
 COPY docker-entrypoint.sh lldap_config.docker_template.toml ./
+COPY example_configs/bootstrap/bootstrap.sh ./
```

Comment:
Be careful, this one is copied in `/app` (see the WORKDIR above) whereas in the other images it's in the root. Maybe let's change the other images to be in /app as well? I don't have a strong opinion, but let's be consistent.

Reply:
OK, I've moved it to `/app` in all the images now in d590a1e3e6e5bd61bbd207557f5e6574a7457921